prow/config.yaml

```diff
@@ -2931,7 +2931,7 @@ periodics:
       - name: GOOGLE_APPLICATION_CREDENTIALS
         value: /etc/service-account/service-account.json
       - name: USER
-        value: prow
+        value: jenkins
```

Comment:
federation pre-submit is already in prow. the problem is the test needs to use the USER explicitely, because federation is bringing up clusters with `${USER}-${zone}`